src/gui/tray/ActivityItem.qml

```diff
@@ -35,6 +31,21 @@ MouseArea {
         color: (parent.containsMouse ? Style.lightHover : "transparent")
     }
 
+    ToolTip {
+        id: activityMouseAreaTooltip
+        visible: containsMouse && !activityContent.childHovered && model.displayLocation !== ""
+        delay: Qt.styleHints.mousePressAndHoldInterval
+        text: qsTr("In %1").arg(model.displayLocation)
+        contentItem: Label {
+            text: activityMouseAreaTooltip.text
+            color: Style.ncTextColor
+        }
+        background: Rectangle {
+            border.color: Style.menuBorder
+            color: Style.backgroundColor
+        }
```

Comment:
if we really need to change the style of every tooltip we ought to build up a real qml style to provide the same customization but without adding more code to every `ToolTip` component

Reply:
> did you had a look at this doc from Qt https://doc.qt.io/qt-5/qml-qtquick-controls2-applicationwindow.html#palette-prop should make it possible to use this to change the colors of controls drawn by Fusion qml style

Yes, though this means we have to go from using a Window to an ApplicationWindow component, I suppose this is not a problem?

Regardless, this does not seem to work as intended -- here are some elements with the custom styling removed:

![image](https://user-images.githubusercontent.com/70155116/153205255-c32ec626-af3c-489f-ae0e-55e13079bea8.png)

Reply:
why did you not choose to use an `ApplicationWindow` in the end ?

Reply:
The palette does not get inherited by child elements, so the change was not needed